spark/components/masthead/styles/_masthead.scss

```
@@ -91,13 +104,12 @@
 // Center logo branding
 .sprk-c-Masthead__branding {
   margin: 0 auto;
+  flex: 2;
+  text-align: center;
 
-  @media (min-width: $sprk-split-breakpoint-xxs) {
-    // Centers logo on narrow by adding space on right to balance menu space on left
-    padding-right: (($sprk-masthead-content-padding + $sprk-masthead-content-item-padding-left + $sprk-masthead-content-item-padding-right) + $sprk-masthead-menu-icon-width);
-  }
   @media (min-width: $sprk-masthead-breakpoint) {
     padding-right: $sprk-masthead-content-item-padding-right-wide;
+    flex-grow:0;
```

```suggestion
    flex-grow: 0;
```
This is probably existing but looks like our linters didn't catch it.